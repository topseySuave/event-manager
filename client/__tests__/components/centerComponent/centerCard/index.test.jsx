import React from 'react';
import { shallow, mount } from 'enzyme';
import CenterCard from
  '../../../../src/components/centerComponent/centerCard/centerCard';

describe('CenterCard component', () => {
  let props = {
    to: '/home',
    center: { id: 3, title: 'never enough', price: '100000' }
  };
  const wrapper = shallow(<CenterCard {...props} />);
  test('component will mount', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
