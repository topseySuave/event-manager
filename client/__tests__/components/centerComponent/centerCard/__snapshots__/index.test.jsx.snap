// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CenterCard component component will mount 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <CenterCard
    center={
      Object {
        "id": 3,
        "price": "100000",
        "title": "never enough",
      }
    }
    to="/home"
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <Link
        href="/home"
        replace={false}
        to="/home"
      >
        <div
          className="card"
        >
          <span
            className="status-indicator darken-3 transparent-status-bar white-text"
          >
            ₦
            100,000
          </span>
          <div
            className="card-image"
          >
            <img
              alt="never enough"
              src={undefined}
            />
          </div>
          <div
            className="card-content black-text"
          >
            <div
              className="row"
              style={
                Object {
                  "marginBottom": "0",
                }
              }
            >
              <div
                className="col s12"
              >
                <p
                  className="bold"
                >
                  never enough
                </p>
                <p
                  className="light__font"
                >
                  <i
                    className="material-icons f15"
                  >
                    location_on
                  </i>
                </p>
              </div>
            </div>
          </div>
          <div
            className="card-action"
          >
            <span
              className="black-text right-align"
            >
              capacity of 
               Guests
            </span>
          </div>
        </div>
      </Link>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": <div
          className="card"
        >
          <span
            className="status-indicator darken-3 transparent-status-bar white-text"
          >
            ₦
            100,000
          </span>
          <div
            className="card-image"
          >
            <img
              alt="never enough"
              src={undefined}
            />
          </div>
          <div
            className="card-content black-text"
          >
            <div
              className="row"
              style={
                Object {
                  "marginBottom": "0",
                }
              }
            >
              <div
                className="col s12"
              >
                <p
                  className="bold"
                >
                  never enough
                </p>
                <p
                  className="light__font"
                >
                  <i
                    className="material-icons f15"
                  >
                    location_on
                  </i>
                </p>
              </div>
            </div>
          </div>
          <div
            className="card-action"
          >
            <span
              className="black-text right-align"
            >
              capacity of 
               Guests
            </span>
          </div>
        </div>,
        "href": "/home",
        "replace": false,
        "to": "/home",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <span
              className="status-indicator darken-3 transparent-status-bar white-text"
            >
              ₦
              100,000
            </span>,
            <div
              className="card-image"
            >
              <img
                alt="never enough"
                src={undefined}
              />
            </div>,
            <div
              className="card-content black-text"
            >
              <div
                className="row"
                style={
                  Object {
                    "marginBottom": "0",
                  }
                }
              >
                <div
                  className="col s12"
                >
                  <p
                    className="bold"
                  >
                    never enough
                  </p>
                  <p
                    className="light__font"
                  >
                    <i
                      className="material-icons f15"
                    >
                      location_on
                    </i>
                  </p>
                </div>
              </div>
            </div>,
            <div
              className="card-action"
            >
              <span
                className="black-text right-align"
              >
                capacity of 
                 Guests
              </span>
            </div>,
          ],
          "className": "card",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                "₦",
                "100,000",
              ],
              "className": "status-indicator darken-3 transparent-status-bar white-text",
            },
            "ref": null,
            "rendered": Array [
              "₦",
              "100,000",
            ],
            "type": "span",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <img
                alt="never enough"
                src={undefined}
              />,
              "className": "card-image",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "alt": "never enough",
                "src": undefined,
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <div
                className="row"
                style={
                  Object {
                    "marginBottom": "0",
                  }
                }
              >
                <div
                  className="col s12"
                >
                  <p
                    className="bold"
                  >
                    never enough
                  </p>
                  <p
                    className="light__font"
                  >
                    <i
                      className="material-icons f15"
                    >
                      location_on
                    </i>
                  </p>
                </div>
              </div>,
              "className": "card-content black-text",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <div
                  className="col s12"
                >
                  <p
                    className="bold"
                  >
                    never enough
                  </p>
                  <p
                    className="light__font"
                  >
                    <i
                      className="material-icons f15"
                    >
                      location_on
                    </i>
                  </p>
                </div>,
                "className": "row",
                "style": Object {
                  "marginBottom": "0",
                },
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <p
                      className="bold"
                    >
                      never enough
                    </p>,
                    <p
                      className="light__font"
                    >
                      <i
                        className="material-icons f15"
                      >
                        location_on
                      </i>
                    </p>,
                  ],
                  "className": "col s12",
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "never enough",
                      "className": "bold",
                    },
                    "ref": null,
                    "rendered": "never enough",
                    "type": "p",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": Array [
                        <i
                          className="material-icons f15"
                        >
                          location_on
                        </i>,
                        undefined,
                      ],
                      "className": "light__font",
                    },
                    "ref": null,
                    "rendered": Array [
                      Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "host",
                        "props": Object {
                          "children": "location_on",
                          "className": "material-icons f15",
                        },
                        "ref": null,
                        "rendered": "location_on",
                        "type": "i",
                      },
                      undefined,
                    ],
                    "type": "p",
                  },
                ],
                "type": "div",
              },
              "type": "div",
            },
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <span
                className="black-text right-align"
              >
                capacity of 
                 Guests
              </span>,
              "className": "card-action",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  "capacity of ",
                  undefined,
                  " Guests",
                ],
                "className": "black-text right-align",
              },
              "ref": null,
              "rendered": Array [
                "capacity of ",
                undefined,
                " Guests",
              ],
              "type": "span",
            },
            "type": "div",
          },
        ],
        "type": "div",
      },
      "type": [Function],
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <Link
          href="/home"
          replace={false}
          to="/home"
        >
          <div
            className="card"
          >
            <span
              className="status-indicator darken-3 transparent-status-bar white-text"
            >
              ₦
              100,000
            </span>
            <div
              className="card-image"
            >
              <img
                alt="never enough"
                src={undefined}
              />
            </div>
            <div
              className="card-content black-text"
            >
              <div
                className="row"
                style={
                  Object {
                    "marginBottom": "0",
                  }
                }
              >
                <div
                  className="col s12"
                >
                  <p
                    className="bold"
                  >
                    never enough
                  </p>
                  <p
                    className="light__font"
                  >
                    <i
                      className="material-icons f15"
                    >
                      location_on
                    </i>
                  </p>
                </div>
              </div>
            </div>
            <div
              className="card-action"
            >
              <span
                className="black-text right-align"
              >
                capacity of 
                 Guests
              </span>
            </div>
          </div>
        </Link>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": <div
            className="card"
          >
            <span
              className="status-indicator darken-3 transparent-status-bar white-text"
            >
              ₦
              100,000
            </span>
            <div
              className="card-image"
            >
              <img
                alt="never enough"
                src={undefined}
              />
            </div>
            <div
              className="card-content black-text"
            >
              <div
                className="row"
                style={
                  Object {
                    "marginBottom": "0",
                  }
                }
              >
                <div
                  className="col s12"
                >
                  <p
                    className="bold"
                  >
                    never enough
                  </p>
                  <p
                    className="light__font"
                  >
                    <i
                      className="material-icons f15"
                    >
                      location_on
                    </i>
                  </p>
                </div>
              </div>
            </div>
            <div
              className="card-action"
            >
              <span
                className="black-text right-align"
              >
                capacity of 
                 Guests
              </span>
            </div>
          </div>,
          "href": "/home",
          "replace": false,
          "to": "/home",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <span
                className="status-indicator darken-3 transparent-status-bar white-text"
              >
                ₦
                100,000
              </span>,
              <div
                className="card-image"
              >
                <img
                  alt="never enough"
                  src={undefined}
                />
              </div>,
              <div
                className="card-content black-text"
              >
                <div
                  className="row"
                  style={
                    Object {
                      "marginBottom": "0",
                    }
                  }
                >
                  <div
                    className="col s12"
                  >
                    <p
                      className="bold"
                    >
                      never enough
                    </p>
                    <p
                      className="light__font"
                    >
                      <i
                        className="material-icons f15"
                      >
                        location_on
                      </i>
                    </p>
                  </div>
                </div>
              </div>,
              <div
                className="card-action"
              >
                <span
                  className="black-text right-align"
                >
                  capacity of 
                   Guests
                </span>
              </div>,
            ],
            "className": "card",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  "₦",
                  "100,000",
                ],
                "className": "status-indicator darken-3 transparent-status-bar white-text",
              },
              "ref": null,
              "rendered": Array [
                "₦",
                "100,000",
              ],
              "type": "span",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <img
                  alt="never enough"
                  src={undefined}
                />,
                "className": "card-image",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "never enough",
                  "src": undefined,
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <div
                  className="row"
                  style={
                    Object {
                      "marginBottom": "0",
                    }
                  }
                >
                  <div
                    className="col s12"
                  >
                    <p
                      className="bold"
                    >
                      never enough
                    </p>
                    <p
                      className="light__font"
                    >
                      <i
                        className="material-icons f15"
                      >
                        location_on
                      </i>
                    </p>
                  </div>
                </div>,
                "className": "card-content black-text",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <div
                    className="col s12"
                  >
                    <p
                      className="bold"
                    >
                      never enough
                    </p>
                    <p
                      className="light__font"
                    >
                      <i
                        className="material-icons f15"
                      >
                        location_on
                      </i>
                    </p>
                  </div>,
                  "className": "row",
                  "style": Object {
                    "marginBottom": "0",
                  },
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      <p
                        className="bold"
                      >
                        never enough
                      </p>,
                      <p
                        className="light__font"
                      >
                        <i
                          className="material-icons f15"
                        >
                          location_on
                        </i>
                      </p>,
                    ],
                    "className": "col s12",
                  },
                  "ref": null,
                  "rendered": Array [
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": "never enough",
                        "className": "bold",
                      },
                      "ref": null,
                      "rendered": "never enough",
                      "type": "p",
                    },
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": Array [
                          <i
                            className="material-icons f15"
                          >
                            location_on
                          </i>,
                          undefined,
                        ],
                        "className": "light__font",
                      },
                      "ref": null,
                      "rendered": Array [
                        Object {
                          "instance": null,
                          "key": undefined,
                          "nodeType": "host",
                          "props": Object {
                            "children": "location_on",
                            "className": "material-icons f15",
                          },
                          "ref": null,
                          "rendered": "location_on",
                          "type": "i",
                        },
                        undefined,
                      ],
                      "type": "p",
                    },
                  ],
                  "type": "div",
                },
                "type": "div",
              },
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <span
                  className="black-text right-align"
                >
                  capacity of 
                   Guests
                </span>,
                "className": "card-action",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    "capacity of ",
                    undefined,
                    " Guests",
                  ],
                  "className": "black-text right-align",
                },
                "ref": null,
                "rendered": Array [
                  "capacity of ",
                  undefined,
                  " Guests",
                ],
                "type": "span",
              },
              "type": "div",
            },
          ],
          "type": "div",
        },
        "type": [Function],
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
